### PR TITLE
Add WorldlineAtosFrankfurt

### PR DESCRIPTION
**Pull request by colinbird**
Adding WorldlineAtosFrankfurt with settings

### CODE REVIEW

File: openapi/components/schemas/GatewayAccountConfig/WorldlineAtosFrankfurt.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+description: WorldlineAtosFrankfurt Gateway config
+allOf:
+  - $ref: ../GatewayAccount.yaml
+  - type: object
```
Comment (slavcodev) [inline openapi/components/schemas/GatewayAccountConfig/WorldlineAtosFrankfurt.yaml]: This is not really needed.

Comment (colinbird) [reply openapi/components/schemas/GatewayAccountConfig/WorldlineAtosFrankfurt.yaml]: Thanks

Comment (colinbird) [inline openapi/components/schemas/GatewayAccountConfig/WorldlineAtosFrankfurt.yaml]:
```suggestion

```

**Comment by adamaltman** [comment]
Aren't there going to be some settings? Like the merchant id?

**Pull request merged**